client/post-editor/editor-delete-post/index.jsx

```diff
@@ -73,7 +77,7 @@ export default React.createClass( {
 		}
 
 		const classes = classnames( 'editor-delete-post__button', { 'is-trashing': this.state.isTrashing } );
-		const label = this.state.isTrashing ? this.translate( 'Trashing...' ) : this.translate( 'Move to trash' );
+		const label = this.state.isTrashing ? this.props.translate( 'Trashing…' ) : this.props.translate( 'Move to trash' );
```

Comment:
:disappointed: 18 existing translations will be lost with this change.
If the string must change, the following option has already been translated **38** times:
`translate( 'Trashing Page' )` *ES Score: 13*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-sites-list-post-actions-and-cache-store)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).